src/style.css

```diff
@@ -0,0 +1,22 @@
+
+body {
+  margin: 0;
+}
+
+.block1 {
+  background-color: red;
+  height: 100px;
+  width: 100%;
+}
+
+.block2 {
+  background-color: green;
+  height: 100px;
+  width: 100%;
+}
+
+.block3 {
+  background-color: blue;
+  height: 100px;
+  width: 100%;
+}
```

Comment:
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.

Reply:
solved